packages/spacebars-compiler/package.js

```diff
@@ -29,14 +29,11 @@ Package.onUse(function (api) {
 
 Package.onTest(function (api) {
   api.use([
-    'underscore@1.0.9',
-    'tinytest@1.0.11',
-    'coffeescript@1.2.4'
-  ]);
-
-  api.use([
+    'underscore',
+    'tinytest',
+    'coffeescript',
     'spacebars-compiler',
-    'blaze-tools@1.0.10'
+    'blaze-tools'
```

Comment:
The reason this is fine is because these are `Package.onTest` dependencies, and we run package tests with a checkout version of Meteor, which has access to local copies of all these packages.